styleguide/screens/Patterns/Calendar/Calendar.js

```diff
@@ -0,0 +1,321 @@
+            The <C>CalendarItem</C> is used to represent a single date.
+          </T>
+
+          <H level={4} className={m.mtLgIi}>Default</H>
+          <T elm="p" className={m.mtr}>
+            By default it shows the current day of the month.
+          </T>
+          <Specimen
+            classNames={{
+              root: m.mtr,
+              specimenContainer: m.par,
+            }}
+            code={dedent`
+              <CalendarItem />
+            `}
+          >
+            <CalendarItem />
+          </Specimen>
+
+          <H level={4} className={m.mtLgIi}>Custom Formatting</H>
+          <T elm="p" className={m.mtr}>
+            The <C>CalendarItem</C> supports different date formats. This can be controlled using
+            the <C>format</C> prop shown below.
+          </T>
+          <Specimen
+            classNames={{
+              root: m.mtr,
+              specimenContainer: m.par,
+            }}
+            code={dedent`
+              <CalendarItem format="MMM" />
+            `}
+          >
+            <CalendarItem format="MMM" />
+          </Specimen>
+
+          <H level={3} className={m.mtLgIi}>Day Range</H>
+          <T elm="p" className={m.mtr}>
+            The Day Range can be used to display a start and an end date. The component is
+            intended to be used a target to date selection. Clicking on either the start date
+            or end date container will indicate the current selection with an underline.
+          </T>
+          <Specimen
+            classNames={{
+              root: m.mtr,
+              specimenContainer: m.par,
+            }}
+            code={dedent`
+              <DayRange
+                startDate={dayRange.startDate}
+                endDate={dayRange.endDate}
+                selectDate={dayRange.selectDate}
+                onStartDateClick={this.handleStartDateClick}
+                onEndDateClick={this.handleEndDateClick}
+               />
+            `}
+          >
+            <DayRange
+              startDate={dayRange.startDate}
+              endDate={dayRange.endDate}
+              selectDate={dayRange.selectDate}
+              onStartDateClick={this.handleStartDateClick}
+              onEndDateClick={this.handleEndDateClick}
+            />
+          </Specimen>
+
+          <H level={3} className={m.mtLgIi}>Calendar Month</H>
+          <T elm="p" className={m.mtr}>
+            The calendar month should be used to represent the current month,
+            the current day is indicated by type weight.
+          </T>
+
+          <H level={4} className={m.mtLgIi}>Default</H>
+          <Specimen
+            classNames={{
+              root: m.mtr,
+              specimenContainer: m.par,
+            }}
+            code={dedent`
+              <CalendarMonth />
+            `}
+          >
+            <CalendarMonth />
+          </Specimen>
+
+          <H level={4} className={m.mtLgIi}>External Dates</H>
```

Comment:
Eternal Dates is a weird title - Out of Range may work better but still not keen.